tests/python/unittest/test_tir_schedule_compute_inline.py

```diff
@@ -185,7 +185,7 @@ def opaque_access_load(a: T.handle, c: T.handle) -> None:
             T.writes(C[0:128, 0:128])
             T.evaluate(
                 T.tvm_access_ptr(
-                    T.type_annotation(dtype="float32"), B.data, 0, 128, "r", dtype="handle"
+                    T.type_annotation(dtype="float32"), B.data, 0, 128, 1, dtype="handle"
```

Comment:
It would be great if we could also handle `"r"` and `"w"` for better readability

Reply:
OK, I use access_prt in this case.:
`  T.evaluate(B.access_ptr("r", extent=128))`